dev-vcs/tortoisehg/tortoisehg-6.1.2.ebuild

```diff
@@ -0,0 +1,81 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{8..10} )
+
+DISTUTILS_USE_SETUPTOOLS=no
```

Comment:
Please use the new `PEP517` mode, docs [here](https://projects.gentoo.org/python/guide/distutils.html).

Reply:
Thank you for review!

Replaced with `DISTUTILS_USE_PEP517=setuptools`.